### ISSUE TITLE
Version 1.3.2 does not support python versions <=3.10

### ISSUE THREAD

**Issue by alexhook** [MKuranowski/aiocsv#28]
Hi!

I have the following exception on python3.10:

```python
ImportError: cannot import name 'Unpack' from 'typing_extensions'
```

According to the typing_extensions [docs](https://typing-extensions.readthedocs.io/en/latest/#typing_extensions.Unpack), the Unpack class has been available since python version 3.11

**Comment by MKuranowski** [comment]
1. aiocsv successfully builds and runs on all versions of CPython ≥ 3.8: https://github.com/MKuranowski/aiocsv/actions/runs/8866966804
2. typing_exceptions docs say that you have linked say that `Unpack` is in pure-Python `typing` starting with 3.11. The whole point of typing_extensions is to backport typing features.
3. The typing_exceptions docs however do say that `Unpack` is available starting with version 4.1.0. Which version of typing_exceptions are you running?

**Comment by alexhook** [comment]
You're right, sorry
